### PR TITLE
Course object

### PR DESCRIPTION
**Pull request by hall**
@ebauman, these changes allow creating scheduled events with courses. It's still a WIP because it sort of kicks scenarios to 2nd class objects as it is (e.g., can't make events with just scenarios :disappointed:) and doesn't allow editing courses in the same way that scenarios can be.

Anyway, just wanted to push it up and get it in front of you and whenever one of us finds the time we can make updates.

### CODE REVIEW

**Comment by ebauman** [comment]
Oof. That's a problem re: ScheduledEvents. 
We'll need to be able to omit those for SEs.

**Comment by hall** [comment]
Definitely. This was a quick and dirty implementation on our end and it just ended up that way because we're not using bare scenarios. I'm aiming to fix it by the month's end.

**Comment by ebauman** [comment]
@hall take a look at these changes, but I think this gets us where we want.

File: src/app/event/new-scheduled-event/new-scheduled-event.component.html [hunk 1]
```diff
@@ -104,6 +135,17 @@
         <clr-dg-cell>{{ s.description }}</clr-dg-cell>
       </clr-dg-row>
     </clr-datagrid>
+
+    <!-- TODO - BUILD THIS AS AN ALERT INSTEAD OF ITALIZED HELPER TEXT.-->
```
Comment (ebauman) [inline src/app/event/new-scheduled-event/new-scheduled-event.component.html]: lol whoops

**Pull request merged**